<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtractFileClass</class>
 <widget class="QMainWindow" name="ExtractFileClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExtractFile</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="SelectBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SelectText">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AppointBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>指定文件夹</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AppointText">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>351</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="ExtractBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>抽取</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>582</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="extractfile.qrc"/>
 </resources>
 <connections/>
</ui>
